sync-core/src/main/java/com/cloudant/sync/datastore/AttachmentManager.java

```diff
@@ -145,6 +149,13 @@ public void addAttachment(SQLDatabase db,PreparedAttachment a, BasicDocumentRevi
 
     }
 
+    public PreparedAttachment createPreparedAttachmentFromAttachment(Attachment att)
```

Comment:
Maybe just `createPreparedAttachment` or `prepareAttachment`? Otherwise it sounds a bit clunky and objc-idiomatic